plugins/dependencies/src/main/resources/refreshVersions-rules/reactivex-version-alias-rules.txt

```diff
@@ -0,0 +1,5 @@
+io.reactivex:*
+             ^
+
+io.reactivex.*:*
+               ^
```

Comment:
Left as is, one would get clashes if a project uses both RxJava 2 and RxJava3, which I believe is likely to happen since the idea behind using different maven coordinates for version 2 and 3 is precisely to allow this.
```suggestion
io.reactivex.*:*
             ^.^
```

Reply:
Gotcha. I don't work with Android/Java/Kotlin as much anymore so I wasn't quite familiar with that. Just wanted to work on adding some deps for my team and the community